bot/exts/info/subscribe.py

```diff
@@ -0,0 +1,149 @@
+import arrow
+import discord
+from discord.ext import commands
+from discord.interactions import Interaction
+
+from bot import constants
+from bot.bot import Bot
+from bot.decorators import in_whitelist
+from bot.log import get_logger
+from bot.utils import checks, members, scheduling
+
+# Tuple of tuples, where each inner tuple is a role id and a month number.
+# The month number signifies what month the role should be assignable,
+# use None for the month number if it should always be active.
+ASSIGNABLE_ROLES = (
```

Comment:
```suggestion
# Tuple of tuples, where each inner tuple is a role id and a month number.
# The month number signifies what month the role should be assignable,
# use None for the month number if it should always be active.
ASSIGNABLE_ROLES: tuple[tuple[int, Optional[int]], ...] = (
```
*note: also requires `from typing import Optional`*

Maybe add an annotation like the comments beautifully describes it?

Reply:
Is there any use in typehinting a constant?

Reply:
Hmm, I see the same argument for making those comments really. But yeah it's not the biggest thing.

Reply:
Yea, the comment is describing what the ints within it signify, whereas plain typehints don't cover that.